its/src/test/java/com/sonar/it/scanner/msbuild/sonarqube/ScannerMSBuildTest.java

```diff
@@ -991,6 +991,39 @@ void incrementalPrAnalysis_ProducesUnchangedFiles() throws IOException {
       .doesNotContain("WithChanges.cs"); // Was modified
   }
 
+  @Test
+  void checkMultiLanguageSupportWithSdkFormat() throws Exception {
+    if (!TestUtils.getMsBuildPath(ORCHESTRATOR).toString().contains("2022")) {
+      return; // This test is not supported on versions older than Visual Studio 22
+    }
+    var folderName = "MultiLanguageSupport";
+    BuildResult result = runBeginBuildAndEndForStandardProject(folderName, "");
+    assertTrue(result.isSuccess());
+
+    // Outside.js is not detected: projectBaseDir is at .csproj level
```

Comment:
One of the easiest improvements that we can make is to include SLQ and PLSQL analysis. Due to this, could you please extend this unit test to reference .SQL files in addition to the .js ones?